### ISSUE TITLE
Calling to_hash method on Ruby models causes a NoMethodError

### ISSUE THREAD

**Issue by sjdemartini** [patch-technology/patch-ruby#5]
### Bug

Calling the `to_hash` method on models (such as `Patch::ProjectListResponse`) leads to a `NoMethodError`, due to ``undefined method `openapi_nullable'``.

### Context
It looks like in this commit https://github.com/patch-technology/patch-ruby/commit/4dc45aff5250a16dcfa208558710c451878efce5#diff-2e1c309b08ff393f103c7a4806b6f7bfL42-L44, the `self.openapi_nullable` methods were (mistakenly?) removed from each of the models, but `openapi_nullable` is still referenced in the `to_hash` methods (e.g. [here](https://github.com/patch-technology/patch-ruby/blob/6dcd2107f95f34167352f11e08b55eb07fe23cb3/lib/patch_ruby/models/allocation.rb#L202)).

### To reproduce
```ruby
require 'patch_ruby'

Patch.configure do |config|
  config.access_token = ENV['PATCH_RUBY_API_KEY']
end

puts Patch::Project.retrieve_projects(page: 1)
```

which should lead to:
```
  5: from app.rb:7:in `puts'
  4: from gems/patch_ruby-1.0.0/lib/patch_ruby/models/project_list_response.rb:212:in `to_s'
  3: from gems/patch_ruby-1.0.0/lib/patch_ruby/models/project_list_response.rb:225:in `to_hash'
  2: from gems/patch_ruby-1.0.0/lib/patch_ruby/models/project_list_response.rb:225:in `each_pair'
  1: from gems/patch_ruby-1.0.0/lib/patch_ruby/models/project_list_response.rb:228:in `block in to_hash'
gems/patch_ruby-1.0.0/lib/patch_ruby/models/project_list_response.rb:52:in `method_missing': undefined method `openapi_nullable' for Patch::ProjectListResponse:Class (NoMethodError)
```

**Comment by biglovisa** [comment]
Thanks for submitting the issue! Apologies for taking so long to get back to you, we have a fix up: #19.

**Comment by sjdemartini** [comment]
Cool!
